src/components/content/order/common/EulaInfo.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import React, { Dispatch, SetStateAction } from 'react';
+import { Checkbox, Form, Space } from 'antd';
+import '../../../../styles/service_order.css';
+import { EulaText } from '../../common/ocl/EulaText';
+import { CheckboxChangeEvent } from 'antd/es/checkbox';
+
+export const EulaInfo = ({
+    eula,
+    isEulaAccepted,
+    setIsEulaAccepted,
+    disabled,
+}: {
+    eula: string | undefined;
+    isEulaAccepted: boolean;
+    setIsEulaAccepted: Dispatch<SetStateAction<boolean>>;
+    disabled: boolean;
+}): React.JSX.Element => {
+    const onChange = (e: CheckboxChangeEvent) => {
+        setIsEulaAccepted(e.target.checked);
+    };
+
+    return (
+        <>
+            {eula && eula.length > 0 ? (
+                <div className={'cloud-provider-tab-class region-flavor-content'}>
+                    <Form.Item
+                        name='isEulaAccepted'
+                        label='Eula'
+                        rules={[{ required: true, message: 'Eula needs to be accepted' }]}
+                    >
+                        <Space wrap>
+                            <Checkbox
+                                checked={isEulaAccepted}
+                                disabled={disabled}
+                                name='isEulaAccepted'
+                                onChange={onChange}
+                            >
+                                I have read and agree to the <EulaText eula={eula} />
```

Comment:
I have read and agreed to the **terms and conditions** of the service. 

and the `terms and conditions` can be a link/hover to the actual EULA text.